src/components/Application.tsx

```diff
@@ -551,10 +573,41 @@ export class Application extends React.Component<Props, State> {
     ])
   }
 
+  //
+  // Inactivity Timeout
+  //
+
+  // Increment the idle time counter every minute.
+  private startIdleTimer() {
+    this.setState({
+      idleTime: 0,
+    })
+
+    this.state.idleInterval = setInterval(this.timerIncrement, 60000)
```

Comment:
Can you bind this to `this` instead of `this.state`?

No telling how React would, err, react, to direct mutations of `this.state` in the future. :trollface: